plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImpl.kt

```diff
@@ -197,12 +197,10 @@ class CameraAnimationsPluginImpl : CameraAnimationsPlugin {
         TAG,
         "Exception while setting camera options : ${e.message} CameraOptions = $cameraOptions"
       )
-    } catch (error: Error) {
-      logE(
-        TAG,
-        "Error while setting camera options : ${error.message} CameraOptions = $cameraOptions"
-      )
     }
+    // notify listeners with actual values
+    notifyListeners(mapCameraManagerDelegate.cameraState)
+    lastCameraOptions = cameraOptions
```

Comment:
either I'm crazy or I do see the same code L192-194

Reply:
nice catch. forgot to remove those lines 🙈